doc/doc-requirements.txt

```diff
@@ -1,4 +1,9 @@
 numpydoc
+mne
```

Comment:
I think we no longer need doc-requirements, they are in the config.yml file now (some with conda, some with pip), WDYT @agramfort ?